Package.swift

```diff
@@ -1,11 +1,15 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.9
```

Comment:
It doesn't seem possible to reference `.visionOS("1.0")` when using `swift-tools-version:5.5`, do we have to split this into separate `Package.swift` and `Package-swift-5.5.swift` files.

Reply:
With some more research I figured out you can use `.custom("visionOS", versionString: "1.0")` for this, which is available in 5.6+. This required dropping support for Swift 5.5, which I think it reasonable at this point.